app/controllers/api/v1/accounts/relationships_controller.rb

```diff
@@ -7,9 +7,7 @@ class Api::V1::Accounts::RelationshipsController < Api::BaseController
   def index
     scope = Account.where(id: account_ids).select('id')
     scope.merge!(Account.without_suspended) unless truthy_param?(:with_suspended)
-    # .where doesn't guarantee that our results are in the same order
-    # we requested them, so return the "right" order to the requestor.
-    @accounts = scope.index_by(&:id).values_at(*account_ids).compact
+    @accounts = scope.compact
```

Comment:
`compact` is not required, I'd remove it altogether or possibly change that to `to_a` if an array is required